validator/src/main/java/io/avaje/validation/core/ResourceBundleManager.java

```diff
@@ -2,17 +2,20 @@
 
 import static java.util.ResourceBundle.getBundle;
 
+import java.lang.System.Logger.Level;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import java.util.MissingResourceException;
 import java.util.ResourceBundle;
 
 import io.avaje.lang.Nullable;
 
 final class ResourceBundleManager {
-
+  private static final System.Logger logger =
+      System.getLogger("io.avaje.validation.core.ResourceBundleManager");
```

Comment:
When running in AWS as lambda, for some unknown reason (seems like AWS bug) the System.Logger is NOT service loaded. This is the motivation for having `AppLog.getLogger()` with 1 extra layer of indirection to service load the logger.

So I think we could update this to use AppLog for that reason.